tools/signing/sign.targets

```diff
@@ -0,0 +1,61 @@
+<Project DefaultTargets="Sign">
+  <Import Project="..\dependencies.props" />
+  <PropertyGroup>
+    <NuGetPackageRoot>$(RestorePackagesPath)</NuGetPackageRoot>
+    <NuGetPackageRoot Condition="!HasTrailingSlash('$(NuGetPackageRoot)')">$(NuGetPackageRoot)\</NuGetPackageRoot>
```

Comment:
Can you please explain why `HasTrailingSlash` check is required here?

Reply:
Just to avoid having a double '\' in the path. I can remove it if desired.